api/src/routing.js

```diff
@@ -397,6 +398,12 @@ app.get('/api/v1/settings', settings.get);
 app.putJson(`${appPrefix}update_settings/${environment.ddoc}`, settings.put); // deprecated
 app.putJson('/api/v1/settings', settings.put);
 
+const purgedDocsController = require('./controllers/purged-docs');
```

Comment:
Can we put this at the top with the other `require` statements?

Reply:
Done